app/api/routes.py

```diff
@@ -42,6 +49,26 @@ def categories():
     return get_categories()
 
 
+@bp.route('/apikey', methods=['POST'], endpoint='apikey')
```

Comment:
Is this a good name for this route? What are the conventions out there?

Reply:
I think the convention is to get an API key from a website, rather than from the API itself. For example, from https://docs.rapidapi.com/docs/keys

![screen shot 2019-02-11 at 8 51 17 pm](https://user-images.githubusercontent.com/2008701/52608248-e49f3580-2e3e-11e9-933e-7a6cd49be9c4.png)

After much searching, I found that Sendgrid calls theirs `/api_key`. See https://sendgrid.api-docs.io/v3.0/api-keys/create-api-keys

Would you like me to add an underscore?